orc8r/cloud/go/sqorc/builder.go

```diff
@@ -318,6 +318,26 @@ func ClearStatementCacheLogOnError(cache *squirrel.StmtCache, callsite string) {
 	}
 }
 
+// FmtConflictUpdateTarget generates the string used to refer to target value
+// used in an on-conflict upsert operation; the specific syntax depends on the
+// sql dialect.
+// e.g. for mariaDB, "f1=t2.f1"; for postgres, "f1=excluded.f1"
```

Comment:
Turn this into a list, making it clear that these are the two options we support. Also, use "t" for table and "c" for column